### DISCUSSION TITLE
Incremental Append mode not working for data stream [global_suppressions] for Sendgrid source

### DISCUSSION THREAD

**Discussion by WisamulHaque** [airbytehq/airbyte#34090] [General]
Description:
In Incremental Append mode for data stream **global_supressions** data is replicating, even when no new data or any changes are done for Sendgrid source.

At destination:
![image](https://github.com/airbytehq/airbyte/assets/106967803/12487932-dca0-4d06-a583-c6e0298c8169)

At source
![image](https://github.com/airbytehq/airbyte/assets/106967803/aa150995-b793-45e2-8646-ea038c2e9a0d)
![image](https://github.com/airbytehq/airbyte/assets/106967803/647773eb-e2ac-4e9c-a940-ba33df5041c8)

Logs:
[SendgridLogs.txt](https://github.com/airbytehq/airbyte/files/13887117/SendgridLogs.txt)

**Comment by sajarin** [comment]
hey @WisamulHaque how many records are being replicated across each sync? This might be [intended behavior ](https://docs.airbyte.com/using-airbyte/core-concepts/sync-modes/incremental-append#overview)

[a comment by an automated account is not shown]
